### PR TITLE
chore: solve high CVE on server docker image

### PR DESCRIPTION
**Pull request by ThibaultFy**
## Description

closes FL-1496

Pass from 32 CVE with 12 high to 8 CVE with 8 medium

## Checklist

- [ ] [changelog](../CHANGELOG.md) was updated with notable changes
- [ ] documentation was updated

### CODE REVIEW

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

**Review by SdgJlbl — APPROVED** [review]
Nicely done 🙇

File: docker/orchestrator-server/Dockerfile [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
-        && rm /tmp/dependencies.json
+    && jq -r '.[] | "\(.name)=\(.version)"' /tmp/dependencies.json | xargs apk add --no-cache \
+    && rm /tmp/dependencies.json
 
 ARG VERSION=dev
 ENV GO111MODULE=on
 ENV SRC_DIR=/usr/src/orchestrator
 
-RUN wget -q https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/v0.4.12/grpc_health_probe-linux-amd64 -O /bin/grpc_health_probe && chmod +x /bin/grpc_health_probe
+RUN wget -q https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/v0.4.25/grpc_health_probe-linux-amd64 -O /bin/grpc_health_probe && chmod +x /bin/grpc_health_probe
```
Comment (guilhem-barthes) [inline docker/orchestrator-server/Dockerfile:17]: 🙏

File: docker/orchestrator-server/Dockerfile [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
 
 # Install migration tool
-RUN wget -qO- https://github.com/golang-migrate/migrate/releases/download/v4.15.2/migrate.linux-amd64.tar.gz | tar xzf - -C /bin/
+RUN wget -qO- https://github.com/golang-migrate/migrate/releases/download/v4.17.0/migrate.linux-amd64.tar.gz | tar xzf - -C /bin/
```
Comment (guilhem-barthes) [inline docker/orchestrator-server/Dockerfile:20]: For later: do we need the migration tool with our compiled binary?

**Pull request merged**